src/http/controllers/controller.auth.ts

```diff
@@ -44,13 +46,54 @@ export module AuthController {
     );
   };
 
+  /**
+   * ! Create new authentication tokens for valid refres tokens
+   * * DanBaDo - 2022/01/14
+   * @param req {Request}
+   * @param res {Response}
+   */
+  export const _refresh = async (req, res) => {
+    const refreshTokenString: string =  req.headers.authorization.split(' ')[1];
+    try {
+      const  refresTokeData = jwt.decode(refreshTokenString,{complete: true});
```

Comment:
You are using the `jwt` library to decode a token, it's not needed, we have a middleware located in `JWTMiddelware` that you need to use to decode a token

Reply:
Function removed in the comming version